src/fontra/client/core/utils.js

```diff
@@ -120,14 +120,18 @@ export function hyphenatedToLabel(s) {
   return capitalizeFirstLetter(s).replaceAll("-", " ");
 }
 
+// platform is deprecated, please see:
+// https://developer.mozilla.org/en-US/docs/Web/API/Navigator/platform
+// export const isMac = typeof navigator !== "undefined" && navigator.platform.toLowerCase().indexOf("mac") >= 0
+
+// Therefore use window.navigator https://developer.mozilla.org/en-US/docs/Web/API/Window/navigator
+export const isMac =
+  typeof navigator !== "undefined" && navigator.userAgent.indexOf("Mac") != -1;
```

Comment:
I don't think we support browsers that don't support `navigator`, so I think that first check is redundant.

Reply:
`npm test` fails without it: _Exception during run: ReferenceError: navigator is not defined_

Reply:
Perfect, makes sense!